Submodule externals/nx_tzdb/tzdb_to_nx contains modified content
Submodule externals/tz/tz contains modified content
diff --git a/externals/nx_tzdb/tzdb_to_nx/externals/tz/tz/Makefile b/externals/nx_tzdb/tzdb_to_nx/externals/tz/tz/Makefile
index 8bee50d8..9c1fe216 100644
--- a/externals/nx_tzdb/tzdb_to_nx/externals/tz/tz/Makefile
+++ b/externals/nx_tzdb/tzdb_to_nx/externals/tz/tz/Makefile
@@ -630,8 +630,7 @@ INSTALL:	ALL install date.1
 # and append "-dirty" if the contents do not already end in "-dirty".
 version:	$(VERSION_DEPS)
 		{ (type git) >/dev/null 2>&1 && \
-		  V=`git describe --match '[0-9][0-9][0-9][0-9][a-z]*' \
-				--abbrev=7 --dirty` || \
+		  V=`echo '2022g-11-g16ce126'` || \
 		  if test '$(VERSION)' = unknown && V=`cat $@`; then \
 		    case $$V in *-dirty);; *) V=$$V-dirty;; esac; \
 		  else \
